copy /etc/main over

nfriedly opened the pull request:
I realized this is one thing we are missing when we use the compiled rootfs.tar. It's not enough on it's own to make the image work, but it's a step in the right direction.

nfriedly committed:
copy /etc/main over

nfriedly commented:
hang on, I think we need something in /etc/inittab also

TriForceX commented:
> hang on, I think we need something in /etc/inittab also

i was about to ask the same

nfriedly committed:
copy /etc/inittab over
this is the one from the rootfs.tar.xz in this repo; it looks a bit different from the one that buildroot generates

nfriedly commented:
Well, I tried starting with the one from the generated by our buildroot repo and modifying it to call /etc/main, and that didn't work, then I tried the one from rootfs.tar.xz in this repo and it also didn't give me anything past the uboot splash, so who knows. We'll go with this one for now, and I'll figure out what else we're missing.
